src/FormidableExtra.res

```diff
@@ -1,15 +1,9 @@
-module ListExtra = {
-  let flatMap = (xs, f) => xs->List.reduce(list{}, (acc, x) => acc->List.concat(f(x)))
-
-  let rec includes = (xs, y) =>
-    switch xs {
-    | list{} => false
-    | list{x, ...xs} => x === y ? true : includes(xs, y)
-    }
+module ArrayExtra = {
+  let flatMap = (xs, f) => xs->Array.reduce([], (acc, x) => acc->Js.Array2.concat(f(x)))
```

Comment:
I wonder why should we use `Js.Array2.concat` instead of `Array.concat` @gaku-sei -san?

Reply:
It's pretty simple, `Js.Array2.concat` is direct bindings to the native `Array.prototype.concat` method, while `Array.concat` is a reimplementation.

We might need to see which is better in the future 😕